src/DotNetWorker.Core/Http/HttpResponseDataExtensions.cs

```diff
@@ -115,7 +158,42 @@ public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T in
         /// <returns>A <see cref="ValueTask"/> that represents the asynchronous operation.</returns>
         public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T instance, ObjectSerializer serializer, CancellationToken cancellationToken = default)
         {
-            return WriteAsJsonAsync(response, instance, serializer, "application/json; charset=utf-8", cancellationToken);
+            return WriteAsJsonAsync(response, instance, serializer, "application/json; charset=utf-8", HttpStatusCode.OK, cancellationToken);
+        }
+
+        /// <summary>
+        /// Write the specified value as JSON to the response body using the provided <see cref="ObjectSerializer"/>.
+        /// The response content-type will be set to <code>application/json; charset=utf-8</code> and the status code set to 200.
```

Comment:
nit: same comment as above about the status code.

Reply:
> Thank you!
> 
> Small nit comments.

Hi @fabiocav , 
i applied your suggestions